### PR TITLE
Make sections active for the return of search

### PR DESCRIPTION
**Pull request by imbstack**
This allows us to be indexed again and cleans up the sidebar nav a bit (to the best of my limited front-end abilities).

### CODE REVIEW

File: src/assets/style.css [hunk 1]
```diff
@@ -63,3 +63,45 @@ nav.navbar-inverse .navbar-text {
...
+  display: block;
+}
+
+ul.toc li a:hover {
+  background-color: #eee;
+}
+
+ul.toc li ul {
+  color: #777;
+  margin-bottom: 1em;
```
Comment (djmitche) [inline src/assets/style.css]: I think this is too large -- it overwhelms the tree shape

Comment (djmitche) [inline src/assets/style.css]: in fact, setting it to 0em still leaves quite a bit of space under each UL, but at least tightens up the space between LIs..

File: src/assets/style.css [hunk 2]
```diff
@@ -63,3 +63,45 @@ nav.navbar-inverse .navbar-text {
...
+}
+
+ul.toc li ul li {
+  color: #777;
+  margin-top: 0.5em;
+  margin-left: 0.5em;
```
Comment (djmitche) [inline src/assets/style.css]: I think this looks a bit better at 1em

**Comment by djmitche** [comment]
CSS is hard.. this looks good, esp the active bit.

**Comment by imbstack** [comment]
Cool, I like those changes as well!

**Pull request merged**